app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -147,26 +147,38 @@ In the following steps, replace `<your-password>` with a secure password.
     ```
 3. Create secret:
     ```
-    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf=<manager-config-filename> --from-file=portal_session_conf=<portal-config-filename>
+    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
     ```
 {% endnavtab %}
 {% endnavtabs %}
 ## Step 7. Prepare Kong's configuration file
 
-1. Create a `values.yaml` file based on the sample in the [Kong charts repository](https://github.com/Kong/charts/blob/main/charts/kong/values.yaml). This file sets all the necessary parameters for your Kong environment.
+1. Create a `values.yaml` file for Helm based on the template in the [Kong charts repository](https://github.com/Kong/charts/blob/main/charts/kong/values.yaml). This file contains all the necessary parameters for your Kong deployment. As an example, [this values file](https://github.com/Kong/charts/blob/main/charts/kong/example-values/full-k4k8s-with-kong-enterprise.yaml) enables most Kong Enterprise features.
```

Comment:
```suggestion
1. Create a `values.yaml` file for Helm based on the template in the [Kong charts repository](https://github.com/Kong/charts/blob/main/charts/kong/values.yaml). This file contains all the possible parameters for your Kong deployment. 

    You can also base your configuration on a sample Kong Enterprise `values.yaml` 
    file. For example, [this values file](https://github.com/Kong/charts/blob/main/charts/kong/example-values/full-k4k8s-with-kong-enterprise.yaml) 
    enables most Kong Enterprise features.
```
I'd suggest a better lead-in for the Kong Enterprise sample file, to make it clear why they might want to use it. 

(I hope I got the indents right, it's difficult to judge in these suggestions.)